introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-transfer-learning-highlevel.ipynb

```diff
@@ -54,8 +54,11 @@
     "print(role)\n",
```

Comment:
<p>Minor: if we're targeting python3 (which we should be) then using f-strings would make the code easier to read.</p><p> </p><p>Same in the rest of the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Ok. I will make the change.</p><p> </p><p>Minor: Format would be inconsistent with other notebooks. But, I assume that is okay. </p>

Reply:
<p>Also, if the notebook was earlier compatible with python, it would be become incompatible now, correct?</p>

Reply:
<p>Is Python2 available in Studio? If not, we should not worry about being compatible with it.</p>

Reply:
<p>List of kernels options do not include python2. So, the answer is no?</p>

Reply:
<p>Python2 is deprecated, and not supported by Studio, apparently. I assume we don't have to be compatible with the outdated version.</p>

Reply:
<p>Sounds good.</p>